frontend/src/components/AcmDataForm.tsx

```diff
@@ -779,8 +784,13 @@ export function AcmDataFormDetails(props: { formData: FormData; wizardSummary?:
                                                       }}
                                                   >
                                                       {section.inputs &&
-                                                          section.inputs.map((input) => (
-                                                              <AcmInputDescription input={input} />
+                                                          section.inputs.map((input, index) => (
+                                                              <AcmInputDescription
+                                                                  input={input}
+                                                                  key={`section--${formSection.title}-${
+                                                                      input.type ?? index
```

Comment:
`formSection.title` and `input.type` are not likely to make a unique combination here. You will often have multiple `Text` inputs in a section. Why not use `input.id`?

Reply:
changed to use id